Sources/Encoder.swift

```diff
@@ -455,9 +455,15 @@ public struct Encoder {
         return {
             url in
             
-            if let url = url {
-                return [key : url.absoluteString]
-            }
+            #if swift(>=2.3)
```

Comment:
Hi! When I was making this change in Swift 2.3 branch, I thought of this solution too and decided against it, because I think you can safety transition to new syntax without the `#if`. Reason is, new code is still compiled correctly and behaves in the same way. The signature simply added one more optional to `url?.absoluteString`, but it doesn't impact the unwrapping.